cmd/server-startup-msg.go

```diff
@@ -67,6 +71,34 @@ func printStartupMessage(apiEndPoints []string) {
 	}
 }
 
+// strip api endpoints list with standard ports such as
+// port "80" and "443" before displaying on the startup
+// banner.  Returns a new list of API endpoints.
+func stripStandardPorts(apiEndpoints []string) (newAPIEndpoints []string) {
+	newAPIEndpoints = make([]string, len(apiEndpoints))
+	// Check all API endpoints for standard ports and strip them.
+	for i, apiEndpoint := range apiEndpoints {
+		url, err := url.Parse(apiEndpoint)
+		if err != nil {
+			newAPIEndpoints[i] = apiEndpoint
+			continue
+		}
+		host, port, _ := net.SplitHostPort(url.Host)
```

Comment:
I don't expect port to exist always, mustSplitHostPort() would crash the server.